### DISCUSSION TITLE
Import Excel Dialog.ts file error

### DISCUSSION THREAD

**Discussion by gfo2007** [serenity-is/Serenity#6548] [Help]
Hi @VictorTomaili (sayuga),
I am trying to experiment with your import functionality. However, my ExcelImportForm class shows errors in my GenreExcelImportDialog.ts class.

The images below show the error. Would be glad if you could point out what I could be doing wrong.

![Excel Import Dialog](https://user-images.githubusercontent.com/14317420/193406285-b9c11b28-6adf-425e-9020-c426f46c8f1a.PNG)
![Excel Import Form](https://user-images.githubusercontent.com/14317420/193406287-3957102b-5247-4c97-8642-2d1f7c10f772.PNG)

Regards,

**Comment by edwardch** [comment]
The TypeScript classes are compiled when you run your project, so first want to check if you have tried to run your project?

**Reply by gfo2007** [reply]
@edwardch I have done that several times. I have even restarted Visual Studio and still nothing changes.

**Comment by arunkumar12712** [comment]
i think restart visual studio after T4, miscellaneous file issue in vs

**Reply by gfo2007** [reply]
@arunkumar12712 done that several times. Yet nothing changed.

**Comment by VictorTomaili** [comment]
The declaration of the form should be like this
https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Web/Modules/Administration/User/UserDialog.ts#L12

And check your application for any error to avoid code generation error. Try to comment current error lines and build successfully your project. Then check your server typings. The FormScriptAttribute should be enough to generate servertypings.
